cocos/renderer/core/gfx/GFXCommandBuffer.h

```diff
@@ -13,7 +13,7 @@ class CC_CORE_API GFXCommandBuffer : public GFXObject {
 public:
   virtual bool initialize(const GFXCommandBufferInfo& info) = 0;
   virtual void destroy() = 0;
-  virtual void begin() = 0;
+  virtual void begin(GFXRenderPass* renderPass = nullptr, uint subpass = 0, GFXFramebuffer* frameBuffer = nullptr) = 0;
```

Comment:
What happened if only some parameters are passed, for example,
```c++
begin(renderPass);
```

Reply:
Then it would mean this secondary command buffer will be executed in the subpass 0 of a renderpass compatible with `renderPass`, with any proper framebuffer.

Reply:
Got it, thanks. But what happened if using like this:
```c++
begin(renderPass, 1);
```

As frame buffer is not passed, but the sub pass is 1. Will the application crash?

Reply:
No, the framebuffer is optional as per vulkan API

Reply:
Good to know, thanks.